monitoring/src/check-page/common-functions.ts

```diff
@@ -46,33 +47,94 @@ export const makeNewBrowser = async (debugMode: boolean): Promise<Browser> => {
 	return browser;
 };
 
+export const openPrivacySettingsPanel = async (config: Config, page: Page) => {
+	log_info(`Loading privacy settings panel: Start`);
+	// Ensure that Sourcepoint has enough time to load the CMP
+	await page.waitForTimeout(3000);
```

Comment:
Is there a way to wait for an element to appear rather than use a timeout?

Reply:
I tried a few alternatives but they all then fail at that point.  Did you have something specific in mind?

Reply:
No, this isn't blocking so go ahead and merge.